solicitdudes/src/app/components/solicitudes/solicitudes.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="filtro">
+  <label class="espacio" for="nombre">Filtrar por nombre: </label>
+  <input id="nombre" [(ngModel)] = "nombre" (keyup)="buscarPorNombre()">
+
+  <label class="espacio" for="email">Filtrar por email: </label>
+  <input id="email" [(ngModel)] = "email" (keyup)="buscarPorEmail()">
+</div>
+@if(solicitudes().length>0){
+  <div>
+    <table class="collectionTable">
+      <tr>
+        <th>Nombre</th>
+        <th>email</th>
+        <th>edad</th>
+        <th (click)="ordenar('experiencia')">Experiencia</th>
+        <th (click)="ordenar('posicion')">Posicion</th>
+        <th>Fecha</th>
+        <th>Estado</th>
+      </tr>
+      @for (item of solicitudes(); track $index) {
```

Comment:
Siempre que se pueda es mejor definir una key única a usar en el `track` del `@for`